wishlist/wishlist.html

```diff
@@ -19,64 +19,52 @@
         <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://neo-ui.netlify.app/styles.css">
         
     </head>
     <body>
-        <nav class="navbar navbar-white bg-white">
-            <p>MyWebsite</p>
+        <nav class="navbar">
+            <p class="head">MyWebsite</p>
             <input class="search" type="text" placeholder="Search"></input>
 
             <div class="nav-right-container">
-                <button class="btn btn-primary">Login</button>
-                <i class="far fa-heart fa-lg"></i><span class="badge1">4</span>
-                <i class="fas fa-shopping-cart fa-lg"></i><span class="badge2">4</span>
+                <button class="login-btn">Login</button>
+                <i class="far fa-heart fa-lg"></i>
+                <i class="fas fa-shopping-cart fa-lg"><span class="badge2">4</span></i>
             </div>
         </nav>
-
         <div class="wishlist">Wishlist</div>
-           <div class="card-container">
-                <div class="card">
-                    <img src="/coat.jpg" alt=""></img>
-                    <i class="fas fa-heart fa-lg"></i>
-                    <label class="img-label">Man Winter Coat</label>
-                    <p class="price">₹600</p>
-                    <button class="btn btn-outline-primary"><a href="/home/index.html">Move to cart</a></button>
-                </div>
-            
-            
+            <div class="grid-container item2">
                 <div class="card">
                     <img src="/coat.jpg" alt=""></img>
-                    <i class="fas fa-heart fa-lg"></i>
+                    <i class="fas fa-heart icon-heart fa-2x"></i>
                     <label class="img-label">Man Winter Coat</label>
-                    <p class="price">₹600</p>
-                    <button class="btn btn-outline-primary"><a href="/home/index.html">Move to cart</a></button>
+                    <p class="price-tag">₹600</p>
+                    <a  class="click-me-link" href="/product/index.html"><button class="move-to-cart">Move to cart</button></a>
                 </div>
-                
                 <div class="card">
                     <img src="/coat.jpg" alt=""></img>
-                    <i class="fas fa-heart fa-lg"></i>
+                    <i class="fas fa-heart icon-heart fa-2x"></i>
                     <label class="img-label">Man Winter Coat</label>
-                    <p class="price">₹600</p>
-                    <button class="btn btn-outline-primary"><a href="/home/index.html">Move to cart</a></button>
+                    <p class="price-tag">₹600</p>
+                    <a  class="click-me-link" href="/product/index.html"><button class="move-to-cart">Move to cart</button></a>
                 </div>
-
                 <div class="card">
                     <img src="/coat.jpg" alt=""></img>
```

Comment:
instead of making these custom cards, use your pre-built component library and make a custom product-card component , it would be easier for you to implement , not that there is something wrong with this.

Reply:
Ok, thanks for the suggestion @ivinayakg